week2/226_invert_binary_tree.py

```diff
@@ -0,0 +1,27 @@
+# https://leetcode.com/problems/invert-binary-tree/
+from typing import Optional
+
+
+class TreeNode:
+    def __init__(self, val=0, left=None, right=None):
+        self.val = val
+        self.left = left
+        self.right = right
+
+
+class Solution:
+    def invert_tree(self, root: Optional[TreeNode]) -> Optional[TreeNode]:
+        if not root:
+            return root
+
+        # DFS - Pre-Order traversal
+        # Post-Order also works - is it faster than Pre-Order?
```

Comment:
There should be no difference because it's a binary tree and the number of nodes you visit are the same.